remoulade/__init__.py

```diff
@@ -90,4 +90,4 @@
     "set_scheduler",
 ]
 
-__version__ = "3.0.0"
+__version__ = "3.0.1"
```

Comment:
do not update this flle and update changelog

Reply:
done